plugin/header/handler.go

```diff
@@ -10,13 +10,16 @@ import (
 
 // Header modifies dns.MsgHdr in the responses
```

Comment:
Comment needs an update.

Reply:
Updated comment. All other comments in `plugin/header` are still valid match with adjusted plugin scope.

Reply:
:+1: